scss/main.scss

```diff
@@ -701,6 +701,22 @@ h1,h2,h3 {
   margin: 0;
 }
 
+// col-md-10
+.iconContainer, .iconTile {
+  position: relative;
+  width: 100%;
+  min-height: 1px;
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+.iconContainer {
+  -webkit-box-flex: 0;
+    -ms-flex: 0 0 83.33333%;
```

Comment:
Nothing to change, but this reminds me that I should get an autoprefixer set up..
Won't need to do this in the future.